python/pyspark/worker.py

```diff
@@ -45,6 +45,8 @@
 
 if sys.version >= '3':
     basestring = str
+else:
+    from itertools import imap as map # use iterator map by default
```

Comment:
two spaces before inlined comment

Reply:
Should I fix it and update the pull request?

Reply:
Yes, you need to fix the style to proceed the jenkins test.